bootstrapvz/providers/ec2/tasks/ami.py

```diff
@@ -125,3 +125,10 @@ def run(cls, info):
             registration_params['sriov_net_support'] = 'simple'
 
         info._ec2['image'] = info._ec2['connection'].register_image(**registration_params)
+
+        # Setting up tags on the AMI
+        if info.manifest.tags:
```

Comment:
when if was failing something was causing whole process to crash, TBH I don't remember now what and why it was like this, it's being rewritten anyway